README.md

```diff
@@ -1,76 +1,55 @@
+<p align="center">
+Jellyfin is a free software media system that puts you in control of managing and streaming your media.
+</p>
 
-Jellyfin is descended from Emby 3.5.2, ported to the .NET Core framework, and aims to contain build facilities for every platform.
-
-For further details, please see [our wiki](https://github.com/jellyfin/jellyfin/wiki). To receive the latest project updates feel free to join [our public chat on Matrix/Riot](https://matrix.to/#/#jellyfin:matrix.org) and to subscribe to [our subreddit](https://www.reddit.com/r/jellyfin/).
-
-## Feature Requests
-
-While our first priority is a stable build, we will eventually add features that were missing in Emby or were not well implemented (technically or philosophically).
-
-[Feature Requests](http://feathub.com/jellyfin/jellyfin)
-
-## Contributing to Jellyfin
-
-If you're interested in contributing, please see [our wiki for guidelines](https://github.com/jellyfin/jellyfin/wiki/Contributing-to-Jellyfin).
-
-## Prebuilt Jellyfin packages
+-----
 
-Prebuild packages are available for Debian/Ubuntu and Arch, and via Docker Hub.
+<p align="center">
+<strong>Want to get started?</strong> Choose from <a href="https://github.com/jellyfin/jellyfin/wiki/Prebuilt-Packages">Prebuilt Packages</a>, or <a href="https://github.com/jellyfin/jellyfin/wiki/Building-from-Source">Build from Source</a>.
+</p>
+<p align="center">
+<strong>Want to contribute?</strong> Check out <a href="https://github.com/jellyfin/jellyfin/wiki/Contributing-to-Jellyfin">our wiki for guidelines</a>.
+</p>
 
-### Docker
+---
 
-The Jellyfin Docker image is available on Docker Hub at https://hub.docker.com/r/jellyfin/jellyfin/
+<p align="center">
+<a href="https://github.com/jellyfin/jellyfin"><img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin.svg"></a>
+<a href="https://cloud.drone.io/jellyfin/jellyfin"><img alt="Build Status" src="https://cloud.drone.io/api/badges/jellyfin/jellyfin/status.svg"></a>
+<a href="https://hub.docker.com/r/jellyfin/jellyfin"><img alt="Docker Pull Count" src="https://img.shields.io/docker/pulls/jellyfin/jellyfin.svg"></a>
+<a href="https://matrix.to/#/#jellyfin:matrix.org"><img alt="Chat on Matrix" src="https://img.shields.io/matrix/!JXmoIxArWgVvbKKqex/matrix.org.svg?logo=matrix"></a>
+<a href="https://www.reddit.com/r/jellyfin/"><img alt="Join our Subreddit" src="https://img.shields.io/badge/reddit-r%2Fjellyfin-%23FF5700.svg"></a>
+</p>
 
-### Arch
+For further details, please see [our wiki](https://github.com/jellyfin/jellyfin/wiki).
 
-The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfin-git/
+To receive the latest project updates feel free to join [our public chat on Matrix/Riot](https://matrix.to/#/#jellyfin:matrix.org), and follow us on [Social Media](https://github.com/jellyfin/jellyfin/wiki/Social-Media).
 
-### Unraid
+## About
```

Comment:
This headline (and "Issues and Feature Requests") does not blend with other headlines which are centered.
IMHO it's better to keep them with unified style - either all left-aligned or all centered (I propose centered - looks nicer to me).

Reply:
I just tried centering the headlines, and it... didn't look right. I think the left-aligned (except for the title) makes sense. I imagine we will eventually add more sections to this, so it will look better in time.